ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -2,34 +2,27 @@
 import keras
 from keras.models import Model
 from keras.layers import Dense, GlobalAveragePooling2D
-from keras.applications.vgg16 import preprocess_input
-from keras.applications.vgg16 import VGG16
+from keras.applications.resnet50 import preprocess_input
+from keras.applications.resnet50 import ResNet50, decode_predictions
 import os
-import rospy
 import tensorflow as tf
 import numpy as np
-import json
 import cv2
-from time import sleep
 
 SPARSE_TO_IDX = {0:0, 1:1, 2:2, 3:4}
-C_TO_COLOR = {0:'RED', 1:'YELLOW', 2:'GREEN', 4: 'UNDEFINED'}
 
 class TLClassifier(object):
     def __init__(self):
         base_path = os.path.dirname(os.path.abspath(__file__))
-        model_file = os.path.join(base_path, 'base_vgg16.h5')
+        model_file = os.path.join(base_path, 'base_rnn50.h5')
 
         self.model = self.create_model()
         self.model.load_weights(model_file)
         self.graph = tf.get_default_graph()
 
 
     def create_model(self):
-        base_model = VGG16(weights='imagenet', include_top=False)
-        for layer in base_model.layers:
-            layer.trainable = False
-                
+        base_model = ResNet50(weights=None, include_top=False)
```

Comment:
no weights are needed ?

Reply:
No, because we don't know does that Carla has internet. Since all layers were retrained - all weights for all layers stored in base_rnn50.h5